src/types/SELECT.ts

```diff
@@ -0,0 +1,5 @@
+export enum SELECT {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
```

Comment:
```suggestion
  ALL = 'all',
  ACTIVE = 'active',
  COMPLETED = 'completed',
```

Reply:
Not fixed 
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/96409563/31043233-cc57-4709-a266-5bc5a4d4ed89)

Reply:
Hello. I can't seem to change it so that the variables are in upper case and the values are in lower case (when iterating dynamically). I also did not find an answer to this question in the chat. allowed to send in this version. I made it to fit the tests. but variables are lowercase. If it's important to you, I can make all these selects non-dynamic. Or you can tell me how to do it correctly